app/controllers/api/v1/notes_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Api
+  module V1
+    class NotesController < ApplicationController
+      def index
+        if valid_type_param
+          render json: notes_filtered, status: :ok, each_serializer: IndexNoteSerializer
+        else
+          render json: { error: I18n.t('note.type_not_allowed') }, status: :not_acceptable
+        end
+      end
+
+      def show
+        render json: show_note, status: :ok, serializer: ShowNoteSerializer
```

Comment:
y a esto con el singular
```suggestion
        render json: note, status: :ok, serializer: ShowNoteSerializer
```